srcpkgs/libXaw3dXft/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libXaw3dXft'
+pkgname=libXaw3dXft
+version=1.6.2h
+revision=1
+build_style=gnu-configure
+configure_args="--disable-static --enable-internationalization --enable-multiplane-bitmaps
+ --enable-gray-stipples --enable-arrow-scrollbars --with-pic --with-gnu-ld"
+hostmakedepends="automake libtool bison pkg-config"
+makedepends="libXft-devel libXmu-devel libXpm-devel xorg-util-macros"
+short_desc="This is an extended version of Xaw3d with support for UTF-8"
+maintainer="xeroxslayer <xeroxslayer@proton.me>"
+license="MIT, X11"
+homepage="https://sourceforge.net/projects/sf-xpaint/files/libxaw3dxft/"
+distfiles="https://sourceforge.net/projects/sf-xpaint/files/libxaw3dxft/${pkgname}-${version}.tar.bz2"
```

Comment:
This isn't that important, but I'd expand `${pkgname}` here.
```suggestion
distfiles="https://sourceforge.net/projects/sf-xpaint/files/libxaw3dxft/libXaw3dXft-${version}.tar.bz2"
```

Reply:
I don't think I follow. The corrected part fits `${pkgname}` perfectly, but other parts of the URL don't.  I should expand this var to also cater to other parts of the URL?